aries-site/src/pages/components/card.mdx

```diff
@@ -14,19 +21,23 @@ import {
   CardWithImage,
   PaginationCardsExample,
   MarketplaceNavigationalCards,
+  NavigationalAnatomy,
   SimpleCard,
   ToolsNavigationalCards,
+  TrialPromotionCard,
+  TrialPromotionRowCard,
+  TrialPromotionColorCard,
 } from '../../examples';
 
 <Example
   componentName="Card"
-  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/card/CardExample.js"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/card/CardKitchenSink.js"
   docs="https://v2.grommet.io/card?theme=hpe#props"
   figma="https://www.figma.com/file/je5AYzMavRt1cG7rmw8j1W/HPE-Card-Component?node-id=1%3A2"
   grommetSource="https://github.com/grommet/grommet/blob/master/src/js/components/Card/Card.js"
+  width="100%"
 >
-
-  <CardExample />
+  <CardKitchenSink />
 </Example>
 
 ## Principles
```

Comment:
A minor visual suggestion...
I'm finding the presentation of these principles a bit heavy. I'm wondering if something like:

- **Contained** - A card is identifiable as a single, contained unit.

might be a bit crisper.